src/pages/groupComparison/Overlap.tsx

```diff
@@ -41,6 +46,27 @@ export default class Overlap extends React.Component<IOverlapProps, {}> {
         this.plotExists = !!this.getSvg();
     }
 
+    readonly tabUI = MakeMobxView({
+        await:()=>{
+            if (this.props.store.activeGroups.isComplete &&
+                this.props.store.activeGroups.result.length < 2) {
+                // dont bother loading data for and computing overlap if not enough groups for it
+                return [this.props.store.activeGroups];
+            } else {
+                return [this.props.store.activeGroups, this.overlapUI];
+            }
+        },
+        render:()=>{
+            if (this.props.store.activeGroups.result!.length < 2) {
+                return <span>{OVERLAP_NOT_ENOUGH_GROUPS_MSG}</span>;
+            } else {
+                return this.overlapUI.component;
+            }
+        },
+        renderPending:()=><Loader isLoading={true} size={"big"}/>,
```

Comment:
I think this should be centered yes? i mean even if we switch to postiion:absolute.

Reply:
yeah, makes sense